Sources/BranchSDK/BNCRequestFactory.m

```diff
@@ -414,6 +414,10 @@ - (void)addAppClipDataToJSON:(NSMutableDictionary *)json {
 - (void)addDefaultRequestDataToJSON:(NSMutableDictionary *)json {
     json[@"branch_key"] = self.branchKey;
     
+    if (self.preferenceHelper.consumerProtectionPreference) {
```

Comment:
Does this get retrieved in time for first open on install?

Reply:
I've confirmed that it does.